openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -34,6 +34,26 @@ export class CourseOutline {  // eslint-disable-line import/prefer-default-expor
       }),
     );
 
+    function expandSection (sectionToggleButton) {
+        const $toggleButtonChevron = $(sectionToggleButton).children('.fa-chevron-right');
```

Comment:
I don't love that line 38 and 39 are the same here and in the collapseSection function, but there doesn't seem to be a nice way to clean that up.